### ISSUE TITLE
Timeout errors when Gatsby is trying to fetch images from Prismic

### ISSUE THREAD

**Issue by einomi** [prismicio/prismic-gatsby#317]
I'm getting timeout errors in the new versions of the plugin (starting 3.0.0-beta.20). 

```
Failed to download https://images.prismic.io/PROJECT_ID/IMAGE_PATH after 3 attempts
```

At first, I was thinking that content managers of the website are uploading too big images and tried to optimize them and to reupload them in Prismic but it turned out that the error happens for small images too. There are 400 images that are fetched from the CMS so it might be a reason of this error. There is an issue on GitHub describing a similar situation with `prismic-javascript` package https://github.com/prismicio/prismic-javascript/issues/44

Also, I've discovered that it only happens for all versions starting 3.0.0-beta.20 (beta 19 works fine). So I had to roll back to 3.0.0-beta.19 for now.

**Comment by My3bIKAHT** [comment]
I keep encountering timeouts even while fetching data from prismic, but only when trying to run builds on mac. No issues on linux. I think there maybe some issue with mac version of node.js

**Comment by MarcMcIntosh** [comment]
Hi @einomi can you share the image url with me?

**Comment by cesarvarela** [comment]
I'm seeing the same issu:

`RequestError: getaddrinfo ENOTFOUND images.prismic.io`

maybe gatbsy is doing too many requests at the same time?

**Comment by einomi** [comment]
> Hi @einomi can you share the image url with me?

Hi @MarcMcIntosh, this is an example URL. It seems it happens for beta 19 too now on my end.
https://images.prismic.io/stillpointspaces/81617611-559f-4d09-8d32-bea81f2c0f20_room+4+sm.jpg?auto=compress,format&rect=0,51,2654,1668&w=1400&h=880

**Comment by michaeljblum** [comment]
So gatsby-plugin-image and Gatsby V3 and generally not yet supported by this plugin? or..

**Comment by angeloashmore** [comment]
Hi @michaeljblum, this plugin works with Gatsby V3 and can you can use gatsby-plugin-image currently if you are downloading images locally with the `localFile` field. If you are using Imgix, we are waiting on the @imgix/gatsby package to export its gatsby-plugin-image functionality.

**Comment by cesarvarela** [comment]
This isn't happening to me anymore and I don't know why.

**Comment by jortizsao** [comment]
> Hi @michaeljblum, this plugin works with Gatsby V3 and you can use gatsby-plugin-image currently if you are downloading images locally with the `localFile` field. If you are using Imgix, we are waiting on the @imgix/gatsby package to export its gatsby-plugin-image functionality to integrate into the next major version of this source plugin.

I think @imgix/gatsby latest version supports the new gatsby-plugin-image https://docs.imgix.com/libraries/gatsby#%E2%9C%A8-new-%E2%9C%A8-gatsbyimage-support

**Comment by angeloashmore** [comment]
Hi @jortizsao, yes, the Imgix package supports gatsby-plugin-image when using it standalone, but it currently does not support integrating directly within other source plugins. That is currently in development so we hope to have it integrated soon.

Until then, gatsby-image is fully integrated. If gatsby-plugin-image is absolutely needed right now, it can be used via the `localFile` field with `gatsby-transformer-sharp` or by using `@imgix/gatsby` to add fields to your documents.

I would recommend waiting for gatsby-plugin-image to be fully integrated into the source plugin before adopting, however.

**Comment by angeloashmore** [comment]
I'm cleaning out old issues and closing this due to its age. The latest version of the plugin supports `gatsby-plugin-image` using `@imgix/gatsby`. If this is still a problem, please open a new issue and I will take a look!
